tests/product/tests/data/process/process_test_cases.py

```diff
@@ -468,18 +468,7 @@
         '/etc/kubernetes/manifests/kube-apiserver.yaml',
         'passed'
     )]
-
 cis_1_2_8 = [(
```

Comment:
why did you remove the test? should we just skip it?

Reply:
The RBAC auth mode for node communication is something that is deprecated from 1.18.
Kind won't support it out of the box. So setting it causes 500 errors from the API server.